clusterloader2/cmd/clusterloader.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"fmt"
+	"k8s.io/perf-tests/clusterloader2/pkg/prometheus"
```

Comment:
nit: Please move this import one block bellow.

Reply:
Done.